lyse/__init__.py

```diff
@@ -102,35 +102,74 @@ def globals_diff(run1, run2, group=None):
     return dict_diff(run1.get_globals(group), run2.get_globals(group))
  
 class Run(object):
+    """A class for saving/retrieving data to/from a shot's hdf5 file.
+
+    This class implements methods that allow the user to retrieve data from a
+    shot's hdf5 file such as images, traces, and the values of globals. It also
+    provides methods for saving and retrieving results from analysis.
+
+    Args:
+        h5_path (str): The path, including file name and extension, to the hdf5
+            file for a shot.
+        no_write (bool, optional): Set to `True` to prevent editing the shot's
+            hdf5 file. Note that doing so prohibits the ability to save results
+            to the file. Defaults to `False`.
+    """
     def __init__(self,h5_path,no_write=False):
-        self.no_write = no_write
-        self._no_group = None
-        self.h5_path = h5_path
+        self._h5_path = h5_path
```

Comment:
These should have double underscores to make them private (but see note on `Sequence` class before changing

Reply:
Updated in 7ba7062 after making the changes suggested below.